pom.xml

```diff
@@ -94,247 +94,8 @@
-    <maven-clean-plugin.version>2.4.1</maven-clean-plugin.version>
-    <maven-compiler-plugin.version>2.3.2</maven-compiler-plugin.version>
-    <maven-dependency-plugin.version>2.1</maven-dependency-plugin.version>
-    <maven-deploy-plugin.version>2.5</maven-deploy-plugin.version>
-    <maven-doap-plugin.version>1.0</maven-doap-plugin.version>
-    <maven-eclipse-plugin.version>2.8</maven-eclipse-plugin.version>
-    <maven-enforcer-plugin.version>1.0</maven-enforcer-plugin.version>
-    <maven-help-plugin.version>2.1.1</maven-help-plugin.version>
-    <maven-install-plugin.version>2.3.1</maven-install-plugin.version>
-    <maven-javadoc-plugin.version>2.7</maven-javadoc-plugin.version>
-    <maven-jar-plugin.version>2.3.1</maven-jar-plugin.version>
-    <maven-jetty-plugin.version>6.1.26</maven-jetty-plugin.version>
-    <maven-jxr-plugin.version>2.2</maven-jxr-plugin.version>
-    <maven-pmd-plugin.version>2.5</maven-pmd-plugin.version>
-    <maven-project-info-reports-plugin.version>2.3.1</maven-project-info-reports-plugin.version>
-    <maven-plugin-plugin.version>2.4.3</maven-plugin-plugin.version>
-    <maven-reactor-plugin.version>1.0</maven-reactor-plugin.version>
-    <maven-release-plugin.version>2.1</maven-release-plugin.version>
-    <maven-remote-resources-plugin.version>1.1</maven-remote-resources-plugin.version>
-    <maven-resources-plugin.version>2.4.3</maven-resources-plugin.version>
-    <maven-site-plugin.version>2.1.1</maven-site-plugin.version>
-    <maven-source-plugin.version>2.1.2</maven-source-plugin.version>
-    <maven-surefire-plugin.version>2.7.2</maven-surefire-plugin.version>
-    <maven-surefire-report-plugin.version>2.7.2</maven-surefire-report-plugin.version>
-    <maven-war-plugin.version>2.1.1</maven-war-plugin.version>
-    <apt-maven-plugin.version>1.0-alpha-4</apt-maven-plugin.version>
-    <axistools-maven-plugin.version>1.4</axistools-maven-plugin.version>
-    <buildnumber-maven-plugin.version>1.0-beta-4</buildnumber-maven-plugin.version>
-    <build-helper-maven-plugin.version>1.5</build-helper-maven-plugin.version>
-    <cargo-maven2-plugin.version>1.0.5</cargo-maven2-plugin.version>
-    <cobertura-maven-plugin.version>2.4</cobertura-maven-plugin.version>
-    <exec-maven-plugin.version>1.2</exec-maven-plugin.version>
-    <findbugs-maven-plugin.version>2.3.1</findbugs-maven-plugin.version>
-    <gwt-maven-plugin.version>2.1.0-1</gwt-maven-plugin.version>
-    <javancss-maven-plugin.version>2.0</javancss-maven-plugin.version>
-    <jdepend-maven-plugin.version>2.0-beta-2</jdepend-maven-plugin.version>
-    <openjpa-maven-plugin.version>1.2</openjpa-maven-plugin.version>
-    <taglist-maven-plugin.version>2.4</taglist-maven-plugin.version>
-    <versions-maven-plugin.version>1.2</versions-maven-plugin.version>
-    <xml-maven-plugin.version>1.0-beta-3</xml-maven-plugin.version>
+    <jenkins.version>1.580.1</jenkins.version>
+    <java.level>6</java.level>
```

Comment:
Minimum supported by `jenkins-test-harness` 2.0. And just a good all-around cutoff point (SECURITY-144 etc.).